pycode/memilio-epidata/memilio/epidata/getCaseData.py

```diff
@@ -46,16 +47,27 @@
 pd.options.mode.copy_on_write = True
 
 
-def check_for_completeness(df, run_checks, merge_berlin=False, merge_eisenach=True):
+def check_for_completeness(df: pd.DataFrame, run_checks: bool, merge_berlin: bool = False, merge_eisenach: bool = True):
     """! Checks if all counties are mentioned in the case data set
 
    This check had to be added due to incomplete data downloads
-   It is checked if all all counties are part of the data.
+   It is checked if all counties are part of the data.
    If data is incomplete the data is downloaded from another source.
    Note: There is no check if data for every day and every county is available (which can happen).
 
-   @param df pandas dataframe to check
-   @return Boolean to say if data is complete or not
+    Parameters
+    ----------
+    df: pd.Dataframe
+        Dataframe to check
+    merge_berlin: bool
+        True or False. Defines if Berlin's districts are kept separated or get merged. Default defined in defaultDict.
+    merge_eisenach: bool
+        True or False. Defines if Eisenbach districts are kept separated or get merged. Default defined in defaultDict.
+    run_checks: bool
+
+    Returns
+    -------
+        Boolean to say if data is complete or not
```

Comment:
This might be nicely formatted but this does not retain our doxygen style comments

Reply:
@mknaranja, I will reformat it.